packages/zipkin-instrumentation-connect/test/integrationTest.js

```diff
@@ -67,26 +52,32 @@ describe('connect instrumentation - integration test', () => {
   }
 
   describe('restify middleware', () => {
+    // Restify uses async hooks. Until there is a CLS hooked implementation here, we need to be
```

Comment:
this was was an oldie but goodie heh. once things are stable we really need that cls hooked thing merged and released!